lisa/tools/dmsetup.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Optional
+
+from lisa.executable import Tool
+
+
+class Dmsetup(Tool):
+    @property
+    def command(self) -> str:
+        return "dmsetup"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def status(self, device_name: str, sudo: bool = True) -> str:
```

Comment:
If some commands are run with sudo always, it doesn't need to provide sudo as a parameter.